examples/gcoap/Makefile

```
@@ -14,17 +14,36 @@ RIOTBASE ?= $(CURDIR)/../..
 USEMODULE += netdev_default
 
 # use GNRC by default
-LWIP ?= 0
+LWIP_IPV4 ?= 0
+LWIP_IPV6 ?= 0
 
-ifeq (0,$(LWIP))
+ifeq (,$(filter 1, $(LWIP_IPV4) $(LWIP_IPV6)))
   USEMODULE += auto_init_gnrc_netif
   # Specify the mandatory networking modules
   USEMODULE += gnrc_ipv6_default
   # Additional networking modules that can be dropped if not needed
   USEMODULE += gnrc_icmpv6_echo
 else
-  USEMODULE += lwip_ipv6
   USEMODULE += lwip_netdev
+
+  ifneq (0,$(LWIP_IPV4))
```

```suggestion
  ifeq (1,$(LWIP_IPV4))
```
is easier to understand

---

Changed